### PR TITLE
Increase loglevel for CI runs

### PR DESCRIPTION
**Pull request by kargakis**
@openshift/sig-azure ptal, part of https://github.com/openshift/openshift-azure/issues/422

### CODE REVIEW

File: pkg/arm/data/master-startup.sh [hunk 1]
```diff
@@ -486,7 +488,11 @@ spec:
     - master
     - api
     - --config=/etc/origin/master/master-config.yaml
+{{- if not .Derived.RunningUnderTest }}
```
Comment (kwoodson) [inline pkg/arm/data/master-startup.sh]: I think I'd prefer the logic swapped on this.  That way the default case is "normal".
```
{{- if .Derived.RunningUnderTest }}
    - --loglevel=4
{{ - else }}
    - --loglevel=2
{{ - end }}
```

Thoughts?

**Comment by jim-minter** [comment]
@kargakis is this a good example of an issue where "@ openshift/sig-azure ptal" might not have been warranted? :)

File: pkg/arm/data/master-startup.sh (hunk 1 above)
Comment (kargakis) [reply pkg/arm/data/master-startup.sh]: sorry, had to repush since I cp'd the above stanza and it has some extra whitespace.

**Pull request merged**